go/teams/transactions.go

```diff
@@ -110,9 +114,8 @@ func (tx *AddMemberTx) createInvite(uv keybase1.UserVersion, role keybase1.TeamR
 	case keybase1.TeamRole_OWNER:
 		payload.Owners = appendToInviteList(invite, payload.Owners)
 	default:
-		return fmt.Errorf("Unexpected role: %v", role)
+		tx.team.G().Log.CWarningf(ctx, "Unexpected role in tx.createInvite(%v, %v)", uv, role)
```

Comment:
Error seems better. Wouldn't want someone to build on top of this and have it surprise no-op.